server/lib/src/utils/flutter_settings.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:appium_flutter_server/src/driver.dart';
+
+Duration getElementWaitTimeout() {
+  if(FlutterDriver.instance.getSession()?.capabilities['flutterElementWaitTimeout'] != null) {
+    return Duration(
+        milliseconds: FlutterDriver.instance.getSession()?.capabilities['flutterElementWaitTimeout']);
+  } else {
+    return const Duration(milliseconds: 5000);
+  }
+}
```

Comment:
This is a great place to start. But eventually we can make it better by maintaining the default states and managing it for every incoming session. I am thinking of something like below

```dart
class FlutterSettings {
   final ELEMENT_WAIT_TIMEOUT_IN_SEC = "flutterElementWaitTimeout"; 
   Map<string, dynamic> _settings = {};
   
   FlutterSettings() {
    setDefaultSetting();
   }
    
   _setDefaultSetting() {
     settings = {
        ELEMENT_WAIT_TIMEOUT_IN_SEC: 5
      };
   }
    
   updateSetting(Map<String, dynamic> capabilities) {
      // if the capability sent by user is present in the settings, then update it. 
      // Else the capability is not known to flutter server and ignore
      capabilities.forEach((settingName, value) => {
        if(_settings[settingName] != null) {
           _settings[settingName] = value;
         }
      });
   }
   
   dynamic getSetting(String settingName) { 
     return _settings[settingName];
   }

   // Called at the end of session
   reset() {
     _setDefaultSetting();
   } 
}
```

And we can access it like,

```
class FlutterDriver {
 //....
 //....
 
 FlutterSettings settings = FlutterSettings();
}
```


`FlutterDriver.instance.settings.get(ELEMENT_WAIT_TIMEOUT_IN_SEC)`